drivers/video/fbdev/core/fbmem.c

```diff
@@ -52,6 +52,7 @@ static DEFINE_MUTEX(registration_lock);
 
 struct fb_info *registered_fb[FB_MAX] __read_mostly;
 int num_registered_fb __read_mostly;
+int min_dynamic_fb;
```

Comment:
This code is paying heavy lip service to optimisation - does `min_dynamic_fb` deserve a `__read_mostly` as well?

Reply:
Done